PyTorch/objectDetection/maskrcnn/requirements.txt

```diff
@@ -0,0 +1,5 @@
+pytorch-directml
+torchvision==0.9.0
```

Comment:
Do we need to include this here?

IIRC the requirements file had some issues with 0.9.0 torchvision